app/frontend/wallet/cardano-ledger-crypto-provider.js

```diff
@@ -0,0 +1,128 @@
+const TransportU2F = require('@ledgerhq/hw-transport-u2f').default // for browser
+const Ada = require('ledgerhq/hw-app-ada').default // temporary hack, should be @ledgerhq
```

Comment:
I would call it "LedgerAda" instead of simply "Ada"

Reply:
Maybe even Ledger would do